gitlab-runner/register-runner.sh

```diff
@@ -1,26 +1,43 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 # gitlab-runner data directory
 DATA_DIR="/etc/gitlab-runner"
 CONFIG_FILE=${CONFIG_FILE:-$DATA_DIR/config.toml}
-# custom certificate authority path
-CA_CERTIFICATES_PATH=${CA_CERTIFICATES_PATH:-$DATA_DIR/certs/ca.crt}
-LOCAL_CA_PATH="/usr/local/share/ca-certificates/ca.crt"
+CACHE_DIR="/cache"
+# Default RUNNER_NAME=HOSTNAME
+RUNNER_NAME=${RUNNER_NAME:-$HOSTNAME}
```

Comment:
RUNNER_NAME should not be used untested. A check if it's defined oder `set -u` should be added.

Reply:
It is tested. If it is not defined then the default is `HOSTNAME`.

Reply:
Set a default for `HOSTNAME` now.